lib/suspenders/app_builder.rb

```diff
@@ -309,6 +309,76 @@ def setup_stylesheets
                 "app/assets/stylesheets/application.scss"
     end
 
+    def install_bourbon_n_friends
+      inject_into_file(
+        'Gemfile',
+        %{\ngem 'bourbon'\ngem 'neat'},
+        after: Regexp.new("gem 'rails', '#{Suspenders::RAILS_VERSION}'")
+      )
+      bundle_command 'install'
+
+      inject_into_file(
+        'app/assets/stylesheets/application.scss',
+        %{\n@import "bourbon";\n@import "neat";},
+        after: /@charset "utf-8";/
+      )
+    end
+
+    def install_bootstrap
+      inject_into_file(
+        'Gemfile',
+        %{\ngem 'bootstrap-sass'},
+        after: /gem 'sass-rails',.*$/
+      )
+      bundle_command 'install'
+
+      inject_into_file(
+        'app/assets/stylesheets/application.scss',
+        %{\n@import "bootstrap-sprockets";\n@import "bootstrap";},
+        after: /@charset "utf-8";/
+      )
+
+      inject_into_file(
+        'app/assets/javascripts/application.js',
+        %{\n\/\/= require bootstrap-sprockets},
+        after: /\/\/= require jquery_ujs/
+      )
+    end
+
+    def install_foundation
+      inject_into_file(
+        'Gemfile',
+        %{\ngem 'foundation-rails'},
+        after: /gem 'sass-rails',.*$/
+      )
+
+      inject_into_file(
+        'app/assets/stylesheets/application.scss',
+        %{\n@import "foundation_and_overrides";},
+        after: /@charset "utf-8";/
+      )
+
+      inject_into_file(
+        'app/assets/javascripts/application.js',
+        %{\n\/\/= require foundation},
```

Comment:
this? https://github.com/philosophie/suspenders/pull/20/files#diff-f274994181e7d14b914a08e06b6167dfR357